flight/Modules/GPS/ubx_cfg.c

```diff
@@ -263,12 +263,22 @@ static void ubx_cfg_poll_version(uintptr_t gps_port) {
 
 //! Apply firmwaree version specific configuration tweaks
 static void ubx_cfg_version_specific(uintptr_t gps_port, uint8_t ver) {
-    if (ver > 6) {
-        // 10Hz for ver 7+
+    if (ver >= 8) {
+        // 10Hz for ver 8
+        // TODO: 5 Hz if using GLONASS
+        ubx_cfg_set_rate(gps_port, (uint16_t)100);
+    } else if (ver == 7) {
+        // 10Hz for ver 7
         ubx_cfg_set_rate(gps_port, (uint16_t)100);
 
-        // SBAS screwed up on v7 modules w/ v1 firmware
-        ubx_cfg_set_sbas(gps_port, 0);
+        // History: Code here used to disable SBAS.  A survey of other projects
```

Comment:
No need to keep this comment or historical code. We have it in git history if needed. Just delete if we are going to.